src/gt4py/testing/suites.py

```diff
@@ -318,7 +357,7 @@ def __new__(cls, cls_name, bases, cls_dict):
 
 
 class StencilTestSuite(metaclass=SuiteMeta):
-    """Base class for every *stencil test suite*.
+    r"""Base class for every *stencil test suite*.
```

Comment:
AFAIK, doctrings cannot be f-strings. Have you verified that raw strings are ok? Does this work fine with sphinx?

Reply:
it's an r-string so the backslash for the underscroe gets passed through to sphinx. it was actually an RST linter error with just the underscore or the backslash, it suggested using the r-string in fact.

Reply:
:+1:

Reply:
actually, that doc string is messed up alltogether. not sure if I should fix this here :laughing: